### ISSUE TITLE
All importers should support option to import without changing data

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#478]
```
It should be possible for users to turn off all unnecessary transformations of 
input data.  Many times these are irreversible, so the time to deal with them 
is at initial import.

One example is the XML importer has no way to turn off string->number 
conversion.  Other converters such strip leading and trailing double quotes 
that they find.
```

Original issue reported on code.google.com by `tfmorris` on 5 Nov 2011 at 4:27
- Blocked on: #308, #402, #420, #578, #596

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Issue 511 has been merged into this issue.
```

Original comment by `tfmorris` on 20 Dec 2011 at 3:44

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 7 Sep 2012 at 9:15
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
In r2451 TabularImportingParserBase now defaults guessCellValueTypes to False 
so that importers which don't specify it don't get it turned on automatically.  
This was adversely affecting Excel, Open Office Calc, and Google Spreadsheets 
because they had no control to turn off the default since they have data types 
built in.
```

Original comment by `tfmorris` on 7 Sep 2012 at 9:24

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 7 Sep 2012 at 9:35
- Now blocked on: #578, #596, #308, #420

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 7 Sep 2012 at 9:35
- Now blocked on: #402

**Comment by GoogleCodeExporter** [comment]
```
I think most of the work is done for this and I'd like to clean up any loose 
ends and get it included in Refine 2.6.
```

Original comment by `tfmorris` on 18 Sep 2012 at 5:52
- Added labels: **Milestone-2.6**
- Now blocked on: #308, #402, #420, #578, #596
